src/mlj/interface.jl

```diff
@@ -73,20 +74,24 @@ function MMI.predict(m::Union{CLF_MODELS...}, (θ, c, classes), Xnew)
     return [MMI.UnivariateFinite(classes, preds[i, :]) for i in 1:size(Xmatrix,1)]
 end
 
-function MMI.fitted_params(m::Union{CLF_MODELS...}, (θ, c, classes))
+function MMI.fitted_params(m::Union{CLF_MODELS...}, (θ, features, c, classes))
     if c > 1
+        W = reshape(θ, :, c)
         if m.fit_intercept
-            W = reshape(θ, div(length(θ), c), c)
-            return (coefs = W, intercept = nothing)
```

Comment:
I _think_ the previous implementation was wrong -- if `m.fit_intercept` is true we should return `intercept`, no ?